sqlite.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
What is the motivation to support sqlite? Doesn't it make sense to move the logic to postgres?

Reply:
In the RootFS I think. But we can add some storage temporarily to the fly.io app and logging all the requests can help us perhaps to debug and in this case, it's better to log them (from a performance pov) in  a local sqlite than in a remote supabase psql db